components/BurgerMenuAndSwitch.js

```diff
@@ -0,0 +1,109 @@
+import {
+  useColorMode,
+  Switch,
+  Flex,
+  Button,
+  IconButton,
+} from "@chakra-ui/react";
+import { useState } from "react";
+import { HamburgerIcon, CloseIcon } from "@chakra-ui/icons";
+import NextLink from "next/link";
+
+export const BurgerMenu = () => {
+  {
+    /*const { colorMode, toggleColorMode } = useColorMode();
+const isDark = colorMode === "dark";
+useless switchmode, just testing*/
+  }
+
```

Comment:
Commented code here. You should delete it. You don't want this in your production code. That's not the droid you're looking for!